src/Management/src/Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -9,42 +8,6 @@ Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.Full = 2 -> Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.None = 0 -> Steeltoe.Management.Configuration.EndpointPermissions
 Steeltoe.Management.Configuration.EndpointPermissions.Restricted = 1 -> Steeltoe.Management.Configuration.EndpointPermissions
-Steeltoe.Management.Diagnostics.DiagnosticObserver
```

Comment:
If we still need `DiagnosticObserver` then I think it (and `IDiagnosticObserver`) should stay in the Abstractions package for extensibility

Reply:
Even if we still need them, they remain public and extensible. But it is only used by the metrics actuator, so close to that seems the better place to me.

Reply:
Keeping it in Abstractions would be better if (as an example) we wanted to add a `DiagnosticObserver` for Eureka. Sure it's only used by the endpoint, but another implementation likely wouldn't be in an application and might not want the heavier dependency on Endpoint